docs/governance/working-groups/Charter-Taxonomy.md

```diff
@@ -0,0 +1,64 @@
+# Project Charter - FINOS CCC Taxonomy Working Group
+
+## Mission
+
+The mission of the FINOS CCC Taxonomy Working Group is to support the aims and ambitions of the FINOS CCC project by creating the taxonomy for the cloud services offered by the cloud hyperscalers.
+
+## Scope
+
+The scope of the project is limited to creating a set of files which capture the offerings for various cloud service offerings in a cloud-agnostic way that focuses on the business value of the service offering rather than its technical implementation. For example, in the case of storage offerings, the taxonomy definition would attempt to capture features that are important to the design of an application such as  availability and durability of the storage rather than the actual implementation of the underlying storage offering.
+
+The scope of the project and the prioritization of effort will be guided and shaped by its contributor members.
+
+## Success Criteria
+
+* Taxonomy files can be used by others
+
+## Deliverables
+
+* One Markdown file per service
+
+## Participation
+
+* Open to all participants
+
+## Roles and Responsibilities
+
+The working group should self-select which of its contributors should have various roles and/or responsibilities assigned to them, e.g., Maintainer or Approver.
+
+## Licensing
+
+* We should use FINOS standard words here
```

Comment:
`Refer to [LICENSE] for more information.`

Reply:
I assume that FINOS *has* standard words to apply here! You're 100% right to refer to them, of course, rather than incorporate them.